evilminions/hydra.py

```diff
@@ -66,6 +70,10 @@ def start(self, hydra_count, chunk, prefix, offset,
 
         io_loop.start()
 
+    def _resolve_slowdown_factors(self, slowdown_factor, random_slowdown_factor, heads_count):
+        random.seed(1)
```

Comment:
You should probably seed this with something hydra-specific, or all "n-th" evil-minions across heads will have the same slowdown